src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md

```diff
@@ -42,3 +42,5 @@ Layout file | Description
 `wishlist_index_configure_type_downloadable.xml` | Layout from this file is applied to `downloadable` product only
 `wishlist_index_configure_type_grouped.xml` | Layout from this file is applied to `grouped` product only
 `wishlist_index_configure_type_simple.xml` | Layout from this file is applied to `simple` product only
+
+You can use custom layout either the design handle for the specific `SKU` or product `ID` or the Layout Update `XM` field in the admin panel for that product.
```

Comment:
Hi @abrarpathan19. This information with examples is already present in the [following](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html#create-cms-pageproductcategory-specific-layouts) topic. If you think that it might be useful for a reader, you can leave a reference here that says something like the following one:

```markdown
For setting a custom layout that depends product id on SKU or ID, check the following topic
```

Reply:
Hello @rogyar I have updated requested changes, please review